src/algorithms/digi/TOFPulseGeneration.cc

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Souvik Paul, Chun Yuen Tsang, Prithwish Tribedy
+// Special Acknowledgement: Kolja Kauder
+//
+// Convert energy deposition into ADC pulses
+// ADC pulses are assumed to follow the shape of landau function
+
+#include <DDRec/CellIDPositionConverter.h>
+#include <RtypesCore.h>
+#include <algorithm>
+#include <gsl/pointers>
+#include <unordered_map>
+#include <vector>
+
+#include "TMath.h"
+#include "TOFPulseGeneration.h"
+#include "algorithms/digi/TOFHitDigiConfig.h"
+
+
+namespace eicrecon {
+
+double TOFPulseGeneration::_Landau(double x, double mean, double std) const {
+  double C = -113.755;
```

Comment:
What's the meaning behind the value of `C` here? Should this be a configurable parameter?

Reply:
Here, _C_ is a parameter that characterizes the gain of a sensor. It has been set based on the realistic gain of a AC-LGAD sensor. It can be set as a configurable parameter, only if the user plans on undertaking studies related to sensor gain.

Reply:
I see! Thanks for the clarification! I think this should definitely be configurable: since this code might be utilized by other subsystems (e.g. one of the MAPS detectors), this number could change...

Reply:
The value is now configurable as the parameter "gain" in TOFHitDigiConfig.h. However, in the current setting, the amplitude of the pulse is fully determined by the parameter "Vm", the amount of energy deposition that corresponds to maximum ADC value. The gain parameter is multiplied to the Landau pulse, but at the end of it when I align the pulse height according to the EDep over Vm ratio, the gain will be divided out. Nonetheless, I will keep this parameter as it maybe useful in the future.